### ISSUE TITLE
Marking forum as members only overrides tag flags with everybody access.

### ISSUE THREAD

**Issue by bwmarrin** [flarum/core#655]
It would be nice (and I would expect) that if you have a private members only forum you could mark a secondary tag or primary tag for public access.  Then posts with those tags would be visible to everyone.  Right now, with the forum base set to members only setting tags for public is ignored.   That, or I'm doing something wrong :)

**Comment by bwmarrin** [comment]
I just tested this a little bit more.  On my system I'm using the latest beta also.

* I set GLOBAL View discussions to Members
* I added a Primary tag "Public Access"
* I set the Public Access tag View discussions to Everyone
* I created a test post with only the Public Access tag
* Logged out

At that point, I could see the Public Access tag listed on the tag list but my test post was not displayed in either the All Discussions list or when I clicked on the Public Access tag.

I didn't test with secondary tags.
